### ISSUE TITLE
Unable to connect SSH, error received from SSH Daemon

### ISSUE THREAD

**Issue by MooseQuest** [gravitational/teleport#2082]
**What happened**:
After setting up through the Quick Start the host in question starts Teleport successfully. However upon connecting (via UI or CLI) the following occurred:

```
ERROR REPORT:
Original Error: *ssh.OpenChannelError ssh: rejected: administratively prohibited (Unable to create identity from connection: access denied to perform action "create" on "user")
Stack Trace:
	/gopath/src/github.com/gravitational/teleport/lib/client/client.go:76 github.com/gravitational/teleport/lib/client.(*ProxyClient).GetSites
	/gopath/src/github.com/gravitational/teleport/lib/client/client.go:667 github.com/gravitational/teleport/lib/client.(*ProxyClient).currentCluster
	/gopath/src/github.com/gravitational/teleport/lib/client/api.go:675 github.com/gravitational/teleport/lib/client.(*TeleportClient).SSH
	/gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:468 main.onSSH
	/gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:265 main.Run
	/gopath/src/github.com/gravitational/teleport/tool/tsh/tsh.go:146 main.main
	/opt/go/src/runtime/proc.go:207 runtime.main
	/opt/go/src/runtime/asm_amd64.s:2362 runtime.goexit
```

This is simply trying to connect to the local server. I suspect there is some issue with DNS.
**What you expected to happen**:
To be presented with bash prompt on the server.
**How to reproduce it (as minimally and precisely as possible)**:

Download Teleport v2.6.5 git:v2.6.5-0-gea40e46b
untar and install via teleport/install
start teleport with -d flag

**Environment**:
- Teleport version: `Teleport v2.6.5 git:v2.6.5-0-gea40e46b`
- Tsh version: `Teleport v2.6.5 git:v2.6.5-0-gea40e46b`
- OS (e.g. from /etc/os-release): `CentOS Linux release 7.5.1804 (Core)`

**Browser environment**

- Browser Version (for UI-related issues): `Chrome Version 67.0.3396.99 (Official Build) (64-bit)`
- Install tools: `via Bash`
- Others:

**Relevant Debug Logs If Applicable**

Let me know if you need anymore logs.

**Comment by MooseQuest** [comment]
As part of my own research, I reviewed the sshd_config of another host to see if there were any glaring issues between the hosts. I've also disabled SELINUX to ensure that I didn't have any issues there. The only thing that I suspect to be the issue is possible a DNS related one. I've configured the host to read the local address, and also the public IP of the host, but that seems to not change anything. I doubt this is a Teleport related issue, but I needed to put "something" out there to possibly get feedback from someone who might know about this particular issue.
